README.md

```diff
@@ -1,25 +1,47 @@
 # Selected Tests
 
-This service is used to predict which tests need to run based on code changes.
+The Selected Tests service is used to predict which tests need to run based on code changes.
 
-## Set up environment
-```
-pip install -r requirements.txt
-pip install -e .
-```
+# Selected Tests API
```

Comment:
I re-ordered the README to put API-related stuff at the top and development mode stuff at the bottom